#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_fix_table_names.dpatch by Stephen Gran <sgran@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct documentation about foreign key references

@DPATCH@
Index: conf/dbschema.sql
===================================================================
--- a/conf/dbschema.sql	(revision 96)
+++ b/conf/dbschema.sql	(working copy)
@@ -26,8 +26,8 @@
 	"gid" int4 NOT NULL,
 	"uid" int4 NOT NULL,
 	PRIMARY KEY ("gid", "uid"),
-	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "groups"("gid"),
-	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "accounts"("uid")
+	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "group_table"("gid"),
+	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "passwd_table"("uid")
 );
   
 CREATE TABLE "shadow_table" (
Index: doc/nss-pgsql.xml
===================================================================
--- a/doc/nss-pgsql.xml	(revision 97)
+++ b/doc/nss-pgsql.xml	(working copy)
@@ -374,8 +374,8 @@
 	"gid" int4 NOT NULL,
 	"uid" int4 NOT NULL,
 	PRIMARY KEY ("gid", "uid"),
-	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "groups"("gid"),
-	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "accounts"("uid")
+	CONSTRAINT "ug_gid_fkey" FOREIGN KEY ("gid") REFERENCES "group_table"("gid"),
+	CONSTRAINT "ug_uid_fkey" FOREIGN KEY ("uid") REFERENCES "passwd_table"("uid")
 );
   
 CREATE TABLE "shadow_table" (
